pom.xml

```diff
@@ -195,30 +215,76 @@ THE SOFTWARE.
             <version>3.5</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.maven.reporting</groupId>
```

Comment:
Why do we need all these Maven dependencies now? Where do we actually use functionality from them?

Reply:
Because of the switch to org.jenkins-ci.plugins 3.0, which defines a very strict maven enforcer configuration. If not met, the build fails. The enforcer rule causing all of this is "[Require Upper Bound Dependencies](https://maven.apache.org/enforcer/enforcer-rules/requireUpperBoundDeps.html)".